### PR TITLE
Random functions

### PR DESCRIPTION
**Pull request by robipresotto**

### CODE REVIEW

File: functions/gm.fish [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function gm
+  git commit -am "$argv"
```
Comment (lukabratos) [inline functions/gm.fish:2]: I think that input argument should be defined as `$argv` without `"` sign.

`git commit -am $argv`

Comment (robipresotto) [inline functions/gm.fish:2]: if we keep the " sign you can type on terminal: gm Commit message
if we take out the " sign you can type on terminal gm "Commit message"

do you like more without " sign?

Comment (lukabratos) [inline functions/gm.fish:2]: Actually we can leave it like you did.

**Comment by lukabratos** [comment]
Thanks @robipresotto 🍻 🎉

**Pull request merged**